### DISCUSSION TITLE
We need a documentation.

### DISCUSSION THREAD

**Discussion by nepalivai** [KRTirtho/spotube#1164] [Ideas]
The discussion is full of similar questions that are asked frequently so I think this repo needs some documentation that answers the questions from the discussion section.

## Solution
I have two ideas for a solution
- add a documentation folder and answer the questions. somehow?
- add documentation on the [site](https://spotube.krtirtho.dev/)

**Comment by KRTirtho** [comment]
Adding FAQ section should be enough. We can add the same thing in the website homepage as well.

**Reply by nepalivai** [reply]
Have any idea on how to get started?

**Reply by KRTirtho** [reply]
We can collect the general usability errors users face and questions that are often asked
